From dc221ba6cd51c23295e721af77ffce3a0dae64fb Mon Sep 17 00:00:00 2001
From: Hansem Ro <hansemro@outlook.com>
Date: Tue, 1 Jun 2021 19:03:43 -0700
Subject: [PATCH] Use system-installed adb

Issue: Saferoot-provided adb is broken on some systems.
Fix: Use system-installed adb instead.

Signed-off-by: Hansem Ro <hansemro@outlook.com>
---
 install.sh | 10 +---------
 unroot.sh  | 10 +---------
 2 files changed, 2 insertions(+), 18 deletions(-)

diff --git a/install.sh b/install.sh
index f937cdc..bf21181 100644
--- a/install.sh
+++ b/install.sh
@@ -52,15 +52,7 @@ chmod 755 files/adb.osx files/adb.linux
 
 systype=$(uname -s)
 
-case x"$systype" in
-	xLinux*)
-		adb="files/adb.linux";;
-	xDarwin*)
-		adb="files/adb.osx";;
-	*)
-		echo "Unrecognized system type $systype. adb must be on your path."
-		adb="adb";;
-esac
+adb="adb"
 
 echo "[*] Testing adb usability"
 echo ""
diff --git a/unroot.sh b/unroot.sh
index 328ce73..5fedf3f 100644
--- a/unroot.sh
+++ b/unroot.sh
@@ -4,15 +4,7 @@ chmod 755 files/adb.osx files/adb.linux
 
 systype=$(uname -s)
 
-case x"$systype" in
-	xLinux*)
-		adb="files/adb.linux";;
-	xDarwin*)
-		adb="files/adb.osx";;
-	*)
-		echo "Unrecognized system type $systype. adb must be on your path."
-		adb="adb";;
-esac
+adb="adb"
 
 echo "[*] Testing adb usability"
 echo ""
-- 
2.31.1

